vpdq/setup.py

```diff
@@ -6,33 +6,55 @@
 import sys
 import subprocess
 from pathlib import Path
+import os
+import logging
 
 DIR = Path(__file__).parent
 read_me = DIR / Path("vpdq/python/README.md")
 long_description = read_me.read_text()
 version = (DIR / "vpdq/version.txt").read_text(encoding="utf-8").strip()
 
+# Get the library directories and include directories from the environment variables
+# These variables should be set in the CMakeLists.txt file
+lib_dirs = os.getenv("LIBRARY_DIRS", "").split(":")
+include_dirs = os.getenv("INCLUDE_DIRS", "").split(":")
+include_dirs.extend(["vpdq/pdq", "./"])
```

Comment:
include_dirs is weird. I couldn't get it to include pdq on Windows for some reason, even though vpdqlib compiles if you manually run the commands.

Something is off with it. I think it works even if the "vpdq/pdq" part is just any random string, but it won't work if it's not there.

Also not sure if "./" should be "." or not.

But on Linux this works fine.

Reply:
Something with environment variable component? 

It behaving better on linux over windows usually has me looking for shell features or file behavior differences. 

I also think this entire file is just brute forcing something that would generate the right output. Watching the original author write this python extension was my first introduction to them, and I tried to learn enough to make some suggestions, but overall I didn't have a good model to pull off of.

Reply:
Yes, it's probably that. LIBRARY_DIRS seemed like it was missing. I'm not worried about it for this PR, but I wanted to document it at the source.